packages/carbon-graphs/tests/unit/controls/Graph/GraphAxes-spec.js

```diff
@@ -2931,6 +2931,102 @@ describe('Graph - Axes', () => {
       });
     });
   });
+  describe('Tick Labels Rotation', () => {
+    fit('tickLabelsRotation values will be 0 or -45', () => {
+      const localeAxisObj = utils.deepClone(axisTimeSeries);
+      localeAxisObj.x = {
+        type: 'timeseries',
+        label: 'Some X Label',
+        lowerLimit: new Date(2017, 0).toISOString(),
+        upperLimit: new Date(2017, 6).toISOString(),
+      };
+      localeAxisObj.x.ticks = {
+        format: '%b %Y',
+        show: true,
+        lowerStepTickValues: [
+          new Date(2017, 1).toISOString(),
+          new Date(2017, 5).toISOString(),
+          new Date(2017, 9).toISOString(),
+        ],
+        midpointTickValues: [
+          new Date(2017, 3).toISOString(),
+          new Date(2017, 7).toISOString(),
+          new Date(2017, 11).toISOString(),
+        ],
+        upperStepTickValues: [
+          new Date(2016, 11).toISOString(),
+          new Date(2018, 1).toISOString(),
+        ],
+        tickLabelsRotation: 0,
+      };
+      graph = new Graph(getAxes(localeAxisObj));
+      expect(localeAxisObj.x.ticks.tickLabelsRotation).toBe(0);
```

Comment:
localeAxisObj is input. We should test for the graphs object in all test cases.
```suggestion
      expect(graph.config.axis.x.ticks.tickLabelsRotation).toBe(0);
```